Mage.Sets/src/mage/cards/g/GenestealerLocus.java

```diff
@@ -0,0 +1,90 @@
+package mage.cards.g;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.AttackedByCreatureTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class GenestealerLocus extends CardImpl {
+
+    public GenestealerLocus(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{U}");
+        this.subtype.add(SubType.TYRANID, SubType.HUMAN);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Neurotraumal Rod — Whenever a creature attacks you, it gets -1/-0 until end of turn.
+        Ability ability = new AttackedByCreatureTriggeredAbility(Zone.BATTLEFIELD,
```

Comment:
you don't need to declare an `Ability` variable, you can just stick the whole thing inside of `addAbility`, also you can remove the zone from the constructor since it's battlefield by default, feel free to change Briar Patch the same way